src/sql/workbench/browser/modelComponents/componentBase.ts

```diff
@@ -274,6 +265,17 @@ export abstract class ComponentBase<TPropertyBag extends azdata.ComponentPropert
 	protected onkeydown(domNode: HTMLElement, listener: (e: StandardKeyboardEvent) => void): void {
 		this._register(addDisposableListener(domNode, EventType.KEY_DOWN, (e: KeyboardEvent) => listener(new StandardKeyboardEvent(e))));
 	}
+
+	protected mergeCss(...styles: azdata.CssStyles[]): azdata.CssStyles {
```

Comment:
This doesn't need to be a part of the class definition

Reply:
I moved it from ContainerBase to ComponentBase so that all components can have access to it.